inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/DataProxyController.java

```diff
@@ -80,4 +82,73 @@ public String getAllConfig(@RequestParam String clusterName, @RequestParam(requi
         return clusterService.getAllConfig(clusterName, md5);
     }
 
+    /**
+     * changeClusterTag
+     */
+    @RequestMapping(value = "/changeClusterTag", method = RequestMethod.POST)
+    @ApiOperation(value = "Change cluster tag and topic of a inlong group id.")
+    public Response<String> changeClusterTag(@RequestBody InlongGroupEntity groupInfo) {
```

Comment:
In the request, it's recommended to use `XxxRequest` rather than `XxxEntity`, because the `XxxEntity` is a persistent object.

Reply:
The parameters change to RequestParam.